src/ray/gcs/gcs_server/gcs_resource_manager.h

```diff
@@ -173,7 +183,8 @@ class GcsResourceManager : public rpc::NodeResourceInfoHandler,
   uint64_t counts_[CountType::CountType_MAX] = {0};
 
   ClusterResourceManager &cluster_resource_manager_;
-  scheduling::NodeID local_node_id_;
+  NodeID local_node_id_;
```

Comment:
What's the reasoning for this change btw? Just for cleaness?

Reply:
Yes. The `scheduling::NodeID` means to be used only in scheduling-related modules.